pkg/sql/catalog/descpb/structured.proto

```diff
@@ -1206,21 +1206,11 @@ message TableDescriptor {
   optional bool schema_locked = 58 [(gogoproto.nullable) = false, (gogoproto.customname) = "SchemaLocked"];
 
   // ImportEpoch is the count of IMPORT INTO jobs that have been attempted on this
-  // table once it __already had data__. The ImportEpoch can include an in progress import, as the
+  // table once. The ImportEpoch can include an in progress import, as the
   // field gets incremented while preparing the table for ingestion.
   optional uint32 import_epoch = 59 [(gogoproto.nullable) = false, (gogoproto.customname) = "ImportEpoch"];
 
-  // ImportType indicates the kind of import into currently occurring on the table. 
-  enum ImportType {
-    NO_IMPORT = 0;
-    IMPORT_INTO_EMPTY = 1;
```

Comment:
are we going to increment an import epoch for IMPORT PGDUMP? If so on restore, I think we need a way to distinguish a table apart of an IMPORT PGDUMP vs an IMPORT INTO. The former should be completely elided from the restore (the table descriptors were never online), while the latter requires a synthetic import job, right?

Reply:
@msbutler 

> are we going to increment an import epoch for IMPORT PGDUMP?

Per the discussion we had: The answer here is "yes for now." Import Pgdump will be excluded in much the same way it is now. We do end up giving up the fast-path rollback for existing-but-empty tables. But as that is pure optimization, we can add it later if we find we really need it.